designs/2021-break-on-parsing-errors/README.md

```diff
@@ -0,0 +1,203 @@
+- Repo: eslint/eslint
+- Start Date: 2021/1/12
+- RFC PR: https://github.com/eslint/rfcs/pull/75
+- Authors: [A-Katopodis](https://github.com/A-Katopodis)
+
+# (Break on parsing errors)
+
+## Summary
+
+The suggested change is for ESLint tp support an argument that will make the CLI exit with code 2 if any fatal errors are reported. The option will be an opt-in boolean argument called `--fatal-parse-error`.
```

Comment:
I think the only open question I have is whether `--fatal-parse-error` makes sense as the flag name. It seems a bit unclear because all parse errors are flagged as fatal, so maybe something like `--exit-on-fatal-error` would better describe the purpose?

Reply:
I agree that `--exit-on-fatal-error` indicates the functionality better. I have updated the RFC to reflect the new name